includes/AMP/Sanitization.php

```diff
@@ -407,7 +404,7 @@ protected function get_sanitizers(): array {
 		$post = get_queried_object();
 
 		if ( $post instanceof \WP_Post && Story_Post_Type::POST_TYPE_SLUG === get_post_type( $post ) ) {
-			$video_cache_enabled = (bool) get_option( Settings::SETTING_NAME_VIDEO_CACHE );
+			$video_cache_enabled = (bool) Services::get( 'settings' )->get_setting( Settings::SETTING_NAME_VIDEO_CACHE );
```

Comment:
Why do it this way and not just inject a class instance?

Reply:
Also, doesn't this make the settings class a dependency of this one? Don't you need to register the setting before this point, to ensure the default values are set.